deploy

```diff
@@ -25,7 +25,10 @@ elif [[ "${TRAVIS_BRANCH}" =~ $ITB_BRANCH_REGEXP ]]; then
     sed -i 's/\(extra_css:\)/\1\n  - css\/itb.css/' mkdocs.yml
     CSS_DIR='docs/css/'
     mkdir -p $CSS_DIR
-    cp ci/itb.css $CSS_DIR
+    # If we set the colors in extra.css, replace them with the ITB colors.
+    # Otherwise, copy over the ITB color CSS.
+    sed -i 's/#343a40/#CC0000/' $CSS_DIR/extra.css || cp ci/itb.css $CSS_DIR
```

Comment:
Pardon my ignorance, but where does `extra.css` come from?  Did you perhaps mean `itb.css` ?

Reply:
[extra.css](https://github.com/search?q=org%3Aopensciencegrid+extra.css&type=Code) is used in many (all?) of the mkdocs repos

Reply:
Ok, i'm not sure i understand what is happening with the `sed ... || cp ...` logic then.

What is the failure condition for `sed` that you want to trigger the `|| cp ...` part?

(Unlike `grep`, `sed` returns success even when no lines are matched/replaced - i would expect it to fail there only if `extra.css` does not exist, or if there is some other io error.)

Reply:
D'oh, I knew that but didn't really think it through. I was just trying to catch the case where `extra.css` didn't exist with the above. 

But if `sed` fails to replace any lines, we still want to copy `itb.css`.